bin/node/src/main.rs

```diff
@@ -89,8 +89,8 @@ async fn main() -> anyhow::Result<()> {
 
     // Open telemetry exporting is only valid for running the node.
     let open_telemetry = match &cli.command {
-        Command::Start { open_telemetry, .. } => *open_telemetry,
-        _ => false,
+        Command::Start { open_telemetry, .. } if *open_telemetry => OpenTelemetry::Enabled,
+        _ => OpenTelemetry::Disabled,
     };
```

Comment:
nitL This looks marginally clearer to me, feel free to disregard
```
let open_telemetry = if let Command::Start { open_telemetry: &true, .. } = &cli.command {
    OpenTelemetry::Enabled
} else {
    OpenTelemetry::Disabled
};
```

Reply:
Ah I never even considered pattern matching inside like that. I do prefer your variant, thank you.